src/lib/features/project/project-owners-read-model.test.ts

```diff
@@ -155,21 +215,132 @@ describe('integration tests', () => {
         });
     });
 
-    test('users are listed before groups', async () => {});
+    test('users are listed before groups', async () => {
+        const projectId = randomId();
+        await db.stores.projectStore.create({ id: projectId, name: projectId });
 
-    test('owners (users and groups) are sorted by when they were added; oldest first', async () => {});
+        await db.stores.accessStore.addGroupToRole(
+            group.id,
+            ownerRoleId,
+            '',
+            projectId,
+        );
 
-    test('returns the system owner for the default project', async () => {});
+        await db.stores.accessStore.addUserToRole(
+            owner.id,
+            ownerRoleId,
+            projectId,
+        );
 
-    test('returns an empty list if there are no projects', async () => {
         const owners = await readModel.getAllProjectOwners();
 
-        expect(owners).toStrictEqual({});
+        expect(owners).toMatchObject({
+            [projectId]: [
+                {
+                    email: 'owner@email.com',
+                    imageUrl: 'image-url-1',
+                    name: 'Owner Name',
+                    ownerType: 'user',
+                },
+                {
+                    name: 'Group Name',
+                    ownerType: 'group',
+                },
+            ],
+        });
+    });
+
+    test('owners (users and groups) are sorted by when they were added; oldest first', async () => {
+        const projectId = randomId();
+        await db.stores.projectStore.create({ id: projectId, name: projectId });
+
+        await db.rawDatabase('role_user').insert({
+            user_id: owner2.id,
+            role_id: ownerRoleId,
+            project: projectId,
+            created_at: new Date('2024-01-01T00:00:00.000Z'),
+        });
+
+        await db.rawDatabase('group_role').insert({
+            group_id: group2.id,
+            role_id: ownerRoleId,
+            project: projectId,
+            created_at: new Date('2024-01-01T00:00:00.000Z'),
+        });
```

Comment:
This one is using the raw database instead of the store to manually add the date, right? Can we mention that in a small comment, just so that the next person doesn't have to wonder?